addons/mass_mailing/models/mail_mail.py

```diff
@@ -3,81 +3,74 @@
 
 import re
 import urlparse
-import re
 import werkzeug.urls
 
-from openerp import tools
-from openerp import SUPERUSER_ID
-from openerp.osv import osv, fields
+from odoo import tools
+from odoo import api, fields, models
```

Comment:
instead of two statement you can write like this : from odoo import api, fields, models, tools